### ISSUE TITLE
Mix deleting unchaged built file

### ISSUE THREAD

**Issue by rodwin** [JeffreyWay/laravel-mix#1107]
- Laravel Mix Version: "laravel-mix": "^0.12.0",
- Node Version (`node -v`): v6.10.2
- NPM Version (`npm -v`): 3.10.10
- OS: windows 10

### Description:
when running watch command initially, my react components and sass are built. Now when I make a change to my sass file, sass is compiled and not the app.js since i didnt made any changes to my app.js but mix deleted the previously built app.js

How to make mix not delete unchage files?

mix config
```
const mix = require('laravel-mix');

const themePath = 'themedir';

mix.react('js/App.js', 'dist/js')
  .scripts([
    `${themePath}/assets/js/OwlCarousel2-2.2.1/dist/owl.carousel.min.js`,
    `${themePath}/assets/js/jquery-match-height/dist/jquery.matchHeight-min.js`,
  ], 'dist/js/vendor.js')
  .sass(`${themePath}/assets/scss/styles.scss`, 'dist/css')
  .sourceMaps()
  .sass(`scss/app.scss`, 'dist/css')
  .copy(`${themePath}/assets/css/fonts.css`, 'dist/css')
  .copyDirectory(`${themePath}/assets/images`, 'dist/images')
  .copyDirectory('images', 'dist/images')
  .copyDirectory(`${themePath}/assets/fonts`, 'dist/fonts');


mix.options({
  processCssUrls: false,
});

// dev environments
if (process.env.NODE_ENV === 'development') {
  // linting
  mix.webpackConfig({
    module: {
      rules: [
        {
          test: /\.(js|jsx)$/,
          exclude: /node_modules/,
          loader: 'eslint-loader',
        },
      ],
    },
  });
}

```
App.js is deleted but vendor.js is not

**Comment by devcircus** [comment]
See #866 
Looks like this was fixed in 1.0

**Comment by rodwin** [comment]
Thanks @devcircus . That fixed my issue.
